src/main.ts

```diff
@@ -1,4 +1,11 @@
 import { bootstrap } from 'ng-metadata/platform';
+import { enableProdMode } from 'ng-metadata/core';
+
 import { AppModule } from './app';
 
+if ('production' === ENV) {
```

Comment:
I prefer `ENV === 'production'` because it is easy to read and it make sense, is 'production' equal to ENV variable value or is ENV variable value equal to 'production'